src/components/main/flight/CarbonEmissionsScreen.js

```diff
@@ -1,235 +1,259 @@
-import React from 'react'
+import React from "react";
```

Comment:
Not checking this screen since it's not part of this ref and will be changed later on. In the future, please leave the reformatting for when you work on the screen since I then have to read through the whole screen to check if anything has changed.